models/resistor.py

```diff
@@ -0,0 +1,10 @@
+class Resistor:
+
```

Comment:
You would want to be consistent with your code style. If you have a blank line before your ```__init__``` method somewhere, either have this blank line in ALL of your classes, or just remove it here.

Reply:
Thanks for mentioning that, will try my best to take care of code convention more precisely and attentively.